enctests/test_configs/color_tests.yml

```
@@ -0,0 +1,174 @@
+
+---
+test_colormatch_raw:
+  app: ffmpeg
+  comparisontest:
+  - testtemplate: idiff {originalfile} {newfile}
+    testtype: idiff
+  - testtype: assertresults
+    tests:
+    - assert: less
+      value: max_error
+      less: 0.00195
+  description: Test from /Users/sam/git/EncodingGuidelines/enctests/../ColorPreservation.md
+    line 44
```

```suggestion
  description: Test from /Users/sam/git/EncodingGuidelines/enctests/../ColorPreservation.md line 44
```
I think yaml needs an indicator if statements go over several lines